# Croatian translation for apport
# Copyright (c) 2006 Rosetta Contributors and Canonical Ltd 2006
# This file is distributed under the same license as the apport package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2006.
#
msgid ""
msgstr ""
"Project-Id-Version: apport\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2014-08-29 11:47+0200\n"
"PO-Revision-Date: 2012-12-27 08:38+0000\n"
"Last-Translator: freedomrun <Unknown>\n"
"Language-Team: Croatian <hr@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2015-02-13 05:40+0000\n"
"X-Generator: Launchpad (build 17336)\n"
"Language: hr\n"

#: ../apport/ui.py:124
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:124
msgid "This package does not seem to be installed correctly"
msgstr "Čini se da ovaj paket nije ispravno instaliran"

#: ../apport/ui.py:128
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:128
#, python-format
msgid ""
"This is not an official %s package. Please remove any third party package "
"and try again."
msgstr ""
"Ovo nije službeni %s paket. Molim vas da uklonite sve nepodržane pakete i "
"pokušate ponovno."

#: ../apport/ui.py:145
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:145
#, python-format
msgid ""
"You have some obsolete package versions installed. Please upgrade the "
"following packages and check if the problem still occurs:\n"
"\n"
"%s"
msgstr ""
"Instalirane su vam neke od zastarjelih verzija paketa. Molim, nadogradite "
"sljedeće pakete i provjerite ponavlja li se problem:\n"
"\n"
"%s"

#: ../apport/ui.py:269
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:269
msgid "unknown program"
msgstr "nepoznati program"

#: ../apport/ui.py:270
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:270
#, python-format
msgid "Sorry, the program \"%s\" closed unexpectedly"
msgstr "Ispričavam se, program \"%s\" se neočekivano zatvorio"

#: ../apport/ui.py:272 ../apport/ui.py:1302
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:272
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1302
#, python-format
msgid "Problem in %s"
msgstr "Problem u %s"

#: ../apport/ui.py:273
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:273
msgid ""
"Your computer does not have enough free memory to automatically analyze the "
"problem and send a report to the developers."
msgstr ""
"Vaše računalo nema dovoljno slobodne memorije za automatsku analizu problema "
"i slanje izvještaja razvojnom timu."

#: ../apport/ui.py:321 ../apport/ui.py:329 ../apport/ui.py:456
#: ../apport/ui.py:459 ../apport/ui.py:660 ../apport/ui.py:1108
#: ../apport/ui.py:1274 ../apport/ui.py:1278
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:321
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:329
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:456
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:459
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:660
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1108
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1274
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1278
msgid "Invalid problem report"
msgstr "Neispravan izvještaj o problemu"

#: ../apport/ui.py:322
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:322
msgid "You are not allowed to access this problem report."
msgstr "Niste ovlašteni za pristup ovom izvještaju o problemu."

#: ../apport/ui.py:325
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:325
msgid "Error"
msgstr "Greška"

#: ../apport/ui.py:326
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:326
msgid "There is not enough disk space available to process this report."
msgstr "Nema dovoljno prostora na disku za obradu ovog izvještaja."

#: ../apport/ui.py:410
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:410
msgid "No package specified"
msgstr "Nema određenih paketa"

#: ../apport/ui.py:411
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:411
msgid ""
"You need to specify a package or a PID. See --help for more information."
msgstr ""
"Morate odrediti paket ili PID. Pogledajte --help za više informacija."

#: ../apport/ui.py:434
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:434
msgid "Permission denied"
msgstr "Dozvola je uskraćena"

#: ../apport/ui.py:435
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:435
msgid ""
"The specified process does not belong to you. Please run this program as the "
"process owner or as root."
msgstr ""
"Određeni proces vam ne pripada. Molimo pokrenite program kao vlasnik procesa "
"ili kao root."

#: ../apport/ui.py:437
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:437
msgid "Invalid PID"
msgstr "Neispravan PID"

#: ../apport/ui.py:438
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:438
msgid "The specified process ID does not belong to a program."
msgstr "Navedeni ID procesa ne pripada programu."

#: ../apport/ui.py:457
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:457
#, python-format
msgid "Symptom script %s did not determine an affected package"
msgstr "Skripta za simptome %s nije odredila pogođen paket"

#: ../apport/ui.py:460
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:460
#, python-format
msgid "Package %s does not exist"
msgstr "Paket %s ne postoji"

#: ../apport/ui.py:484 ../apport/ui.py:672 ../apport/ui.py:677
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:484
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:672
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:677
msgid "Cannot create report"
msgstr "Ne mogu napraviti izvještaj"

#: ../apport/ui.py:499 ../apport/ui.py:545 ../apport/ui.py:562
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:499
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:545
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:562
msgid "Updating problem report"
msgstr "Ažuriranje izvještaja o problemu"

#: ../apport/ui.py:500
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:500
msgid ""
"You are not the reporter or subscriber of this problem report, or the report "
"is a duplicate or already closed.\n"
"\n"
"Please create a new report using \"apport-bug\"."
msgstr ""
"Niste autor ili pretplatnik izvještaja ovog problema, moguće da je izvještaj "
"duplikat ili je već zatvoren.\n"
"\n"
"Molim napravite novi izvještaj koristeći \"apport-bug\"."

#: ../apport/ui.py:509
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:509
msgid ""
"You are not the reporter of this problem report. It is much easier to mark a "
"bug as a duplicate of another than to move your comments and attachments to "
"a new bug.\n"
"\n"
"Subsequently, we recommend that you file a new bug report using \"apport-"
"bug\" and make a comment in this bug about the one you file.\n"
"\n"
"Do you really want to proceed?"
msgstr ""

#: ../apport/ui.py:546 ../apport/ui.py:563
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:546
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:563
msgid "No additional information collected."
msgstr "Nisu prikupljene dodatne informacije."

#: ../apport/ui.py:614
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:614
msgid "What kind of problem do you want to report?"
msgstr "Kakav problem želite prijaviti?"

#: ../apport/ui.py:631
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:631
msgid "Unknown symptom"
msgstr "Nepoznat simptom"

#: ../apport/ui.py:632
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:632
#, python-format
msgid "The symptom \"%s\" is not known."
msgstr "Simptom \"%s\" nije poznat."

#: ../apport/ui.py:663
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:663
msgid ""
"After closing this message please click on an application window to report a "
"problem about it."
msgstr ""
"Nakon zatvaranja ove poruke, molim kliknite na prozor aplikacije za prijavu "
"problema o ovome."

#: ../apport/ui.py:673 ../apport/ui.py:678
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:673
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:678
msgid "xprop failed to determine process ID of the window"
msgstr "xprop nije uspio odrediti proces ID prozora"

#: ../apport/ui.py:692
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:692
msgid "%prog <report number>"
msgstr "%prog <broj izvještaja>"

#: ../apport/ui.py:694
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:694
msgid "Specify package name."
msgstr "Odredite ime paketa."

#: ../apport/ui.py:696 ../apport/ui.py:747
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:696
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:747
msgid "Add an extra tag to the report. Can be specified multiple times."
msgstr "Dodajte dodatnu oznaku u izvještaju. Moguće je odrediti više puta."

#: ../apport/ui.py:726
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:726
msgid ""
"%prog [options] [symptom|pid|package|program path|.apport/.crash file]"
msgstr ""
"%prog [options] [symptom|pid|package|program path|.apport/.crash file]"

#: ../apport/ui.py:729
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:729
msgid ""
"Start in bug filing mode. Requires --package and an optional --pid, or just "
"a --pid. If neither is given, display a list of known symptoms. (Implied if "
"a single argument is given.)"
msgstr ""

#: ../apport/ui.py:731
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:731
msgid "Click a window as a target for filing a problem report."
msgstr "Kliknite na prozor kao cilj za podnošenje izvještaja o problemu."

#: ../apport/ui.py:733
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:733
msgid "Start in bug updating mode. Can take an optional --package."
msgstr ""
"Početak načina za bug ažuriranje. Opcionalno se može dodati --package."

#: ../apport/ui.py:735
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:735
msgid ""
"File a bug report about a symptom. (Implied if symptom name is given as only "
"argument.)"
msgstr ""

#: ../apport/ui.py:737
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:737
msgid ""
"Specify package name in --file-bug mode. This is optional if a --pid is "
"specified. (Implied if package name is given as only argument.)"
msgstr ""
"Navedite ime paketa u --file-bug načinu. Ovo je opcija ukoliko je --pid "
"naveden. (Podrazumijeva se ako je ime paketa navedeno kao jedini argument.)"

#: ../apport/ui.py:739
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:739
msgid ""
"Specify a running program in --file-bug mode. If this is specified, the bug "
"report will contain more information.  (Implied if pid is given as only "
"argument.)"
msgstr ""

#: ../apport/ui.py:741
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:741
msgid "The provided pid is a hanging application."
msgstr ""

#: ../apport/ui.py:743
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:743
#, python-format
msgid ""
"Report the crash from given .apport or .crash file instead of the pending "
"ones in %s. (Implied if file is given as only argument.)"
msgstr ""

#: ../apport/ui.py:745
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:745
msgid ""
"In bug filing mode, save the collected information into a file instead of "
"reporting it. This file can then be reported later on from a different "
"machine."
msgstr ""

#: ../apport/ui.py:749
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:749
msgid "Print the Apport version number."
msgstr "Ispis broja Apport inačice."

#: ../apport/ui.py:888
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:888
msgid ""
"This will launch apport-retrace in a terminal window to examine the crash."
msgstr ""
"To će pokrenuti apport-retrace u prozoru terminala za ispitivanje kraha."

#: ../apport/ui.py:889
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:889
msgid "Run gdb session"
msgstr "Pokreni gdb sesiju"

#: ../apport/ui.py:890
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:890
msgid "Run gdb session without downloading debug symbols"
msgstr "Pokreni gdb sesiju bez preuzimanja simbola za ispravljanje pogrešaka"

#. TRANSLATORS: %s contains the crash report file name
#: ../apport/ui.py:892
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:892
#, python-format
msgid "Update %s with fully symbolic stack trace"
msgstr "Ažuriranje %s sa punim simboličkim stack tragovima"

#: ../apport/ui.py:968 ../apport/ui.py:978
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:968
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:978
msgid ""
"This problem report applies to a program which is not installed any more."
msgstr ""

#: ../apport/ui.py:993
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:993
#, python-format
msgid ""
"The problem happened with the program %s which changed since the crash "
"occurred."
msgstr ""
"Došlo je do problema s programom %s koji se promijenio otkako da se srušio."

#: ../apport/ui.py:1040 ../apport/ui.py:1280
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1040
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1280
msgid "This problem report is damaged and cannot be processed."
msgstr "Ovaj izvještaj o problemu je oštećen i ne može biti obrađen."

#. package does not exist
#: ../apport/ui.py:1043
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1043
msgid "The report belongs to a package that is not installed."
msgstr "Izvještaj pripada paketu koji nije instaliran."

#: ../apport/ui.py:1046
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1046
msgid "An error occurred while attempting to process this problem report:"
msgstr "Došlo je do greške pri pokušaju obrade izvještaja o problemu:"

#: ../apport/ui.py:1109
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1109
msgid "Could not determine the package or source package name."
msgstr "Ne može se odrediti paket ili naziv izvornog paketa."

#: ../apport/ui.py:1127
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1127
msgid "Unable to start web browser"
msgstr "Nije moguće pokrenuti internet preglednik"

#: ../apport/ui.py:1128
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1128
#, python-format
msgid "Unable to start web browser to open %s."
msgstr "Nije moguće pokrenuti internet preglednik za otvaranje %s."

#: ../apport/ui.py:1228
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1228
#, python-format
msgid "Please enter your account information for the %s bug tracking system"
msgstr ""

#: ../apport/ui.py:1240
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1240
msgid "Network problem"
msgstr "Problem s mrežom"

#: ../apport/ui.py:1242
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1242
msgid ""
"Cannot connect to crash database, please check your Internet connection."
msgstr ""

#: ../apport/ui.py:1269
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1269
msgid "Memory exhaustion"
msgstr "Nedostatak memorije"

#: ../apport/ui.py:1270
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1270
msgid "Your system does not have enough memory to process this crash report."
msgstr ""
"Vaš sustav nema dovoljno memorije za obradu ovog izvještaja o rušenju "
"sustava."

#: ../apport/ui.py:1305
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1305
#, python-format
msgid ""
"The problem cannot be reported:\n"
"\n"
"%s"
msgstr ""
"Ne može se prijaviti ovaj problem:\n"
"\n"
"%s"

#: ../apport/ui.py:1361 ../apport/ui.py:1368
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1361
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1368
msgid "Problem already known"
msgstr "Problem je već poznat"

#: ../apport/ui.py:1362
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1362
msgid ""
"This problem was already reported in the bug report displayed in the web "
"browser. Please check if you can add any further information that might be "
"helpful for the developers."
msgstr ""
"Ovaj problem je već prijavljen u izvještaju grešaka koji je prikazan u "
"internet pregledniku. Molim provjerite možete li dodati neke dodatne "
"informacije koje mogu pomoći razvojnom timu."

#: ../apport/ui.py:1369
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1369
msgid "This problem was already reported to developers. Thank you!"
msgstr ""

#: ../debian/tmp/usr/bin/apport-cli.py:74 ../bin/apport-cli.py:74
msgid "Press any key to continue..."
msgstr "Pritisnite bilo koju tipku za nastavak..."

#: ../debian/tmp/usr/bin/apport-cli.py:81 ../bin/apport-cli.py:81
msgid "What would you like to do? Your options are:"
msgstr "Što želite napraviti? Vaše mogućnosti su:"

#: ../debian/tmp/usr/bin/apport-cli.py:85 ../bin/apport-cli.py:85
#, python-format
msgid "Please choose (%s):"
msgstr "Molim odaberite (%s):"

#: ../debian/tmp/usr/bin/apport-cli.py:148 ../bin/apport-cli.py:148
#, python-format
msgid "(%i bytes)"
msgstr "(%i bajtova)"

#: ../kde/apport-kde.py:348 ../gtk/apport-gtk.py:142
#: ../debian/tmp/usr/share/apport/apport-kde.py:348
#: ../debian/tmp/usr/bin/apport-cli.py:150 ../bin/apport-cli.py:150
#: ../debian/tmp/usr/share/apport/apport-gtk.py:142
msgid "(binary data)"
msgstr "(binarni podatci)"

#: ../kde/apport-kde.py:171 ../gtk/apport-gtk.py:177
#: ../debian/tmp/usr/share/apport/apport-kde.py:171
#: ../debian/tmp/usr/bin/apport-cli.py:178 ../bin/apport-cli.py:178
#: ../debian/tmp/usr/share/apport/apport-gtk.py:177
msgid "Send problem report to the developers?"
msgstr "Pošaljite izvještaj o problemu razvojnom timu?"

#: ../debian/tmp/usr/bin/apport-cli.py:179 ../bin/apport-cli.py:179
msgid ""
"After the problem report has been sent, please fill out the form in the\n"
"automatically opened web browser."
msgstr ""
"Nakon slanja izvještaja o problemu, molimo ispunite formular u\n"
"automatski otvorenom internet pregledniku."

#: ../debian/tmp/usr/bin/apport-cli.py:182 ../bin/apport-cli.py:182
#, python-format
msgid "&Send report (%s)"
msgstr "&Pošaljite izvještaj (%s)"

#: ../debian/tmp/usr/bin/apport-cli.py:186 ../bin/apport-cli.py:186
msgid "&Examine locally"
msgstr ""

#: ../debian/tmp/usr/bin/apport-cli.py:190 ../bin/apport-cli.py:190
msgid "&View report"
msgstr "&Pogledajte izvještaj"

#: ../debian/tmp/usr/bin/apport-cli.py:191 ../bin/apport-cli.py:191
msgid "&Keep report file for sending later or copying to somewhere else"
msgstr ""
"&Zadržite dokument izvještaja za kasnije slanje ili kopiranje na neko drugo "
"mjesto"

#: ../debian/tmp/usr/bin/apport-cli.py:192 ../bin/apport-cli.py:192
msgid "Cancel and &ignore future crashes of this program version"
msgstr "Poništi i &ignoriraj buduća rušenja ove verzije programa"

#: ../debian/tmp/usr/bin/apport-cli.py:194
#: ../debian/tmp/usr/bin/apport-cli.py:271
#: ../debian/tmp/usr/bin/apport-cli.py:303
#: ../debian/tmp/usr/bin/apport-cli.py:324 ../bin/apport-cli.py:194
#: ../bin/apport-cli.py:271 ../bin/apport-cli.py:303 ../bin/apport-cli.py:324
msgid "&Cancel"
msgstr "&Odustani"

#: ../debian/tmp/usr/bin/apport-cli.py:222 ../bin/apport-cli.py:222
msgid "Problem report file:"
msgstr "Dokument izvještaja o problemu:"

#: ../debian/tmp/usr/bin/apport-cli.py:228
#: ../debian/tmp/usr/bin/apport-cli.py:233 ../bin/apport-cli.py:228
#: ../bin/apport-cli.py:233
msgid "&Confirm"
msgstr "&Potvrdi"

#: ../debian/tmp/usr/bin/apport-cli.py:232 ../bin/apport-cli.py:232
#, python-format
msgid "Error: %s"
msgstr "Greška: %s"

#: ../kde/apport-kde.py:392 ../debian/tmp/usr/share/apport/apport-kde.py:392
#: ../debian/tmp/usr/bin/apport-cli.py:238 ../bin/apport-cli.py:238
msgid "Collecting problem information"
msgstr "Skupljanje informacija o problemu"

#: ../debian/tmp/usr/bin/apport-cli.py:239 ../bin/apport-cli.py:239
msgid ""
"The collected information can be sent to the developers to improve the\n"
"application. This might take a few minutes."
msgstr ""
"Prikupljene informacije mogu biti poslane razvojnom timu kako bi se "
"aplikacija\n"
"poboljšala. To može potrajati nekoliko minuta."

#: ../kde/apport-kde.py:420 ../debian/tmp/usr/share/apport/apport-kde.py:420
#: ../debian/tmp/usr/bin/apport-cli.py:251 ../bin/apport-cli.py:251
#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:15 ../gtk/apport-gtk.ui.h:15
msgid "Uploading problem information"
msgstr "Slanje informacija o problemu"

#: ../debian/tmp/usr/bin/apport-cli.py:252 ../bin/apport-cli.py:252
msgid ""
"The collected information is being sent to the bug tracking system.\n"
"This might take a few minutes."
msgstr ""
"Prikupljene informacije se šalju sustavu za praćenje grešaka.\n"
"Ovo može potrajati nekoliko minuta."

#: ../debian/tmp/usr/bin/apport-cli.py:302 ../bin/apport-cli.py:302
msgid "&Done"
msgstr "&Obavljeno"

#: ../debian/tmp/usr/bin/apport-cli.py:308 ../bin/apport-cli.py:308
msgid "none"
msgstr "nijedno"

#: ../debian/tmp/usr/bin/apport-cli.py:309 ../bin/apport-cli.py:309
#, python-format
msgid "Selected: %s. Multiple choices:"
msgstr "Označeno: %s. Višestruki izbori:"

#: ../debian/tmp/usr/bin/apport-cli.py:325 ../bin/apport-cli.py:325
msgid "Choices:"
msgstr "Izbori:"

#: ../debian/tmp/usr/bin/apport-cli.py:339 ../bin/apport-cli.py:339
msgid "Path to file (Enter to cancel):"
msgstr "Putanja do datoteke ( Enter za prekid):"

#: ../debian/tmp/usr/bin/apport-cli.py:345 ../bin/apport-cli.py:345
msgid "File does not exist."
msgstr "Datoteka ne postoji."

#: ../debian/tmp/usr/bin/apport-cli.py:347 ../bin/apport-cli.py:347
msgid "This is a directory."
msgstr "Ovo je direktorij."

#: ../debian/tmp/usr/bin/apport-cli.py:353 ../bin/apport-cli.py:353
msgid "To continue, you must visit the following URL:"
msgstr "Da biste nastavili, morate posjetiti sljedeći URL:"

#: ../debian/tmp/usr/bin/apport-cli.py:355 ../bin/apport-cli.py:355
msgid ""
"You can launch a browser now, or copy this URL into a browser on another "
"computer."
msgstr ""
"Sada možete pokrenuti pretreživač, ili kopirati URL u pretreživač na drugom "
"računalu."

#: ../debian/tmp/usr/bin/apport-cli.py:357 ../bin/apport-cli.py:357
msgid "Launch a browser now"
msgstr "Sada pokrenite preglednik"

#: ../debian/tmp/usr/bin/apport-cli.py:371 ../bin/apport-cli.py:371
msgid "No pending crash reports. Try --help for more information."
msgstr ""
"Nema neposlanih izvještaja o rušenju. Pokušajte --help za više informacija."

#: ../debian/tmp/usr/share/apport/apportcheckresume.py:73
#: ../data/apportcheckresume.py:73
msgid ""
"This occured during a previous suspend and prevented it from resuming "
"properly."
msgstr ""

#: ../debian/tmp/usr/share/apport/apportcheckresume.py:75
#: ../data/apportcheckresume.py:75
msgid ""
"This occured during a previous hibernate and prevented it from resuming "
"properly."
msgstr ""

#: ../debian/tmp/usr/share/apport/apportcheckresume.py:80
#: ../data/apportcheckresume.py:80
msgid ""
"The resume processing hung very near the end and will have appeared to have "
"completed normally."
msgstr ""

#: ../debian/tmp/usr/bin/apport-retrace.py:34 ../bin/apport-retrace.py:34
msgid "Do not put the new traces into the report, but write them to stdout."
msgstr ""

#: ../debian/tmp/usr/bin/apport-retrace.py:36 ../bin/apport-retrace.py:36
msgid ""
"Start an interactive gdb session with the report's core dump (-o ignored; "
"does not rewrite report)"
msgstr ""

#: ../debian/tmp/usr/bin/apport-retrace.py:38 ../bin/apport-retrace.py:38
msgid ""
"Write modified report to given file instead of changing the original report"
msgstr ""

#: ../debian/tmp/usr/bin/apport-retrace.py:41 ../bin/apport-retrace.py:41
msgid "Remove the core dump from the report after stack trace regeneration"
msgstr ""

#: ../debian/tmp/usr/bin/apport-retrace.py:43 ../bin/apport-retrace.py:43
msgid "Override report's CoreFile"
msgstr ""

#: ../debian/tmp/usr/bin/apport-retrace.py:45 ../bin/apport-retrace.py:45
msgid "Override report's ExecutablePath"
msgstr ""

#: ../debian/tmp/usr/bin/apport-retrace.py:47 ../bin/apport-retrace.py:47
msgid "Override report's ProcMaps"
msgstr ""

#: ../debian/tmp/usr/bin/apport-retrace.py:49 ../bin/apport-retrace.py:49
msgid "Rebuild report's Package information"
msgstr "Obnovi izvješća o informacijama paketa."

#: ../debian/tmp/usr/bin/apport-retrace.py:51 ../bin/apport-retrace.py:51
msgid ""
"Build a temporary sandbox and download/install the necessary packages and "
"debug symbols in there; without this option it assumes that the necessary "
"packages and debug symbols are already installed in the system. The argument "
"points to the packaging system configuration base directory; if you specify "
"\"system\", it will use the system configuration files, but will then only "
"be able to retrace crashes that happened on the currently running release."
msgstr ""

#: ../debian/tmp/usr/bin/apport-retrace.py:53 ../bin/apport-retrace.py:53
msgid ""
"Report download/install progress when installing packages into sandbox"
msgstr ""

#: ../debian/tmp/usr/bin/apport-retrace.py:55 ../bin/apport-retrace.py:55
msgid "Prepend timestamps to log messages, for batch operation"
msgstr ""

#: ../debian/tmp/usr/bin/apport-retrace.py:57 ../bin/apport-retrace.py:57
msgid "Cache directory for packages downloaded in the sandbox"
msgstr ""

#: ../debian/tmp/usr/bin/apport-retrace.py:59 ../bin/apport-retrace.py:59
msgid ""
"Directory for unpacked packages. Future runs will assume that any already "
"downloaded package is also extracted to this sandbox."
msgstr ""

#: ../debian/tmp/usr/bin/apport-retrace.py:61
#: ../debian/tmp/usr/bin/apport-valgrind.py:66 ../bin/apport-valgrind.py:66
#: ../bin/apport-retrace.py:61
msgid ""
"Install an extra package into the sandbox (can be specified multiple times)"
msgstr ""

#: ../debian/tmp/usr/bin/apport-retrace.py:63 ../bin/apport-retrace.py:63
msgid ""
"Path to a file with the crash database authentication information. This is "
"used when specifying a crash ID to upload the retraced stack traces (only if "
"neither -g, -o, nor -s are specified)"
msgstr ""

#: ../debian/tmp/usr/bin/apport-retrace.py:65 ../bin/apport-retrace.py:65
msgid ""
"Display retraced stack traces and ask for confirmation before sending them "
"to the crash database."
msgstr ""

#: ../debian/tmp/usr/bin/apport-retrace.py:67 ../bin/apport-retrace.py:67
msgid ""
"Path to the duplicate sqlite database (default: no duplicate checking)"
msgstr ""

#: ../debian/tmp/usr/bin/apport-retrace.py:76 ../bin/apport-retrace.py:76
msgid "You cannot use -C without -S. Stopping."
msgstr ""

#. translators: don't translate y/n, apport currently only checks for "y"
#: ../debian/tmp/usr/bin/apport-retrace.py:109 ../bin/apport-retrace.py:109
msgid "OK to send these as attachments? [y/n]"
msgstr "Jeli u redu otposlati ove privitke? [d/n]"

#: ../kde/apport-kde.desktop.in.h:1 ../gtk/apport-gtk.desktop.in.h:1
#: ../kde/apport-kde-mime.desktop.in.h:1
msgid "Report a problem..."
msgstr "Prijavite problem..."

#: ../kde/apport-kde.desktop.in.h:2 ../gtk/apport-gtk.desktop.in.h:2
#: ../kde/apport-kde-mime.desktop.in.h:2
msgid "Report a malfunction to the developers"
msgstr "Prijavite kvar razvojnom timu"

#: ../debian/tmp/usr/share/apport/kernel_oops.py:29 ../data/kernel_oops.py:29
msgid "Your system might become unstablenow and might need to be restarted."
msgstr ""

#: ../gtk/apport-gtk.py:157 ../debian/tmp/usr/share/apport/apport-gtk.py:157
#, python-format
msgid "Sorry, the application %s has stopped unexpectedly."
msgstr ""

#: ../gtk/apport-gtk.py:160 ../debian/tmp/usr/share/apport/apport-gtk.py:160
#, python-format
msgid "Sorry, %s has closed unexpectedly."
msgstr ""

#: ../kde/apport-kde.py:185 ../kde/apport-kde.py:223 ../gtk/apport-gtk.py:165
#: ../debian/tmp/usr/share/apport/apport-kde.py:185
#: ../debian/tmp/usr/share/apport/apport-kde.py:223
#: ../debian/tmp/usr/share/apport/apport-gtk.py:165
#, python-format
msgid "Sorry, %s has experienced an internal error."
msgstr ""

#: ../kde/apport-kde.py:179 ../gtk/apport-gtk.py:186
#: ../debian/tmp/usr/share/apport/apport-kde.py:179
#: ../debian/tmp/usr/share/apport/apport-gtk.py:186
msgid "Send"
msgstr ""

#: ../kde/apport-kde.py:277 ../gtk/apport-gtk.py:204 ../gtk/apport-gtk.py:573
#: ../debian/tmp/usr/share/apport/apport-kde.py:277
#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:9 ../gtk/apport-gtk.ui.h:9
#: ../debian/tmp/usr/share/apport/apport-gtk.py:204
#: ../debian/tmp/usr/share/apport/apport-gtk.py:573
msgid "Show Details"
msgstr ""

#: ../kde/apport-kde.py:216 ../kde/apport-kde.py:229 ../gtk/apport-gtk.py:216
#: ../gtk/apport-gtk.py:284 ../gtk/apport-gtk.py:303
#: ../debian/tmp/usr/share/apport/apport-kde.py:216
#: ../debian/tmp/usr/share/apport/apport-kde.py:229
#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:12 ../gtk/apport-gtk.ui.h:12
#: ../debian/tmp/usr/share/apport/apport-gtk.py:216
#: ../debian/tmp/usr/share/apport/apport-gtk.py:284
#: ../debian/tmp/usr/share/apport/apport-gtk.py:303
msgid "Continue"
msgstr ""

#: ../gtk/apport-gtk.py:228 ../debian/tmp/usr/share/apport/apport-gtk.py:228
msgid "Force Closed"
msgstr ""

#: ../kde/apport-kde.py:220 ../kde/apport-kde.py:366 ../gtk/apport-gtk.py:229
#: ../gtk/apport-gtk.py:288 ../debian/tmp/usr/share/apport/apport-kde.py:220
#: ../debian/tmp/usr/share/apport/apport-kde.py:366
#: ../debian/tmp/usr/share/apport/apport-gtk.py:229
#: ../debian/tmp/usr/share/apport/apport-gtk.py:288
msgid "Relaunch"
msgstr ""

#: ../gtk/apport-gtk.py:238 ../debian/tmp/usr/share/apport/apport-gtk.py:238
#, python-format
msgid "The application %s has stopped responding."
msgstr ""

#: ../gtk/apport-gtk.py:242 ../debian/tmp/usr/share/apport/apport-gtk.py:242
#, python-format
msgid "The program \"%s\" has stopped responding."
msgstr ""

#: ../kde/apport-kde.py:193 ../gtk/apport-gtk.py:257
#: ../debian/tmp/usr/share/apport/apport-kde.py:193
#: ../debian/tmp/usr/share/apport/apport-gtk.py:257
#, python-format
msgid "Package: %s"
msgstr ""

#: ../kde/apport-kde.py:199 ../gtk/apport-gtk.py:264
#: ../debian/tmp/usr/share/apport/apport-kde.py:199
#: ../debian/tmp/usr/share/apport/apport-gtk.py:264
msgid "Sorry, a problem occurred while installing software."
msgstr ""

#: ../kde/apport-kde.py:205 ../gtk/apport-gtk.py:273 ../gtk/apport-gtk.py:292
#: ../debian/tmp/usr/share/apport/apport-kde.py:205
#: ../debian/tmp/usr/share/apport/apport-gtk.py:273
#: ../debian/tmp/usr/share/apport/apport-gtk.py:292
#, python-format
msgid "The application %s has experienced an internal error."
msgstr ""

#: ../kde/apport-kde.py:208 ../gtk/apport-gtk.py:276
#: ../debian/tmp/usr/share/apport/apport-kde.py:208
#: ../debian/tmp/usr/share/apport/apport-gtk.py:276
#, python-format
msgid "The application %s has closed unexpectedly."
msgstr ""

#: ../kde/apport-kde.py:219 ../gtk/apport-gtk.py:287
#: ../debian/tmp/usr/share/apport/apport-kde.py:219
#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:11 ../gtk/apport-gtk.ui.h:11
#: ../debian/tmp/usr/share/apport/apport-gtk.py:287
msgid "Leave Closed"
msgstr ""

#: ../kde/apport-kde.py:226 ../gtk/apport-gtk.py:300
#: ../debian/tmp/usr/share/apport/apport-kde.py:226
#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:6 ../gtk/apport-gtk.ui.h:6
#: ../debian/tmp/usr/share/apport/apport-gtk.py:300
msgid "If you notice further problems, try restarting the computer."
msgstr ""

#: ../kde/apport-kde.py:230 ../gtk/apport-gtk.py:304
#: ../debian/tmp/usr/share/apport/apport-kde.py:230
#: ../debian/tmp/usr/share/apport/apport-gtk.py:304
msgid "Ignore future problems of this type"
msgstr ""

#: ../kde/apport-kde.py:274 ../gtk/apport-gtk.py:577
#: ../debian/tmp/usr/share/apport/apport-kde.py:274
#: ../debian/tmp/usr/share/apport/apport-gtk.py:577
msgid "Hide Details"
msgstr ""

#: ../debian/tmp/usr/bin/apport-valgrind.py:37 ../bin/apport-valgrind.py:37
msgid "See man page for details."
msgstr ""

#: ../debian/tmp/usr/bin/apport-valgrind.py:43 ../bin/apport-valgrind.py:43
msgid "specify the log file name produced by valgrind"
msgstr ""

#: ../debian/tmp/usr/bin/apport-valgrind.py:46 ../bin/apport-valgrind.py:46
msgid ""
"reuse a previously created sandbox dir (SDIR) or, if it does not exist, "
"create it"
msgstr ""

#: ../debian/tmp/usr/bin/apport-valgrind.py:50 ../bin/apport-valgrind.py:50
msgid ""
"do  not  create  or reuse a sandbox directory for additional debug symbols "
"but rely only on installed debug symbols."
msgstr ""

#: ../debian/tmp/usr/bin/apport-valgrind.py:54 ../bin/apport-valgrind.py:54
msgid ""
"reuse a previously created cache dir (CDIR) or, if it does not exist, create "
"it"
msgstr ""

#: ../debian/tmp/usr/bin/apport-valgrind.py:58 ../bin/apport-valgrind.py:58
msgid ""
"report download/install progress when installing packages into sandbox"
msgstr ""

#: ../debian/tmp/usr/bin/apport-valgrind.py:62 ../bin/apport-valgrind.py:62
msgid ""
"the executable that is run under valgrind's memcheck tool  for memory leak "
"detection"
msgstr ""

#: ../debian/tmp/usr/bin/apport-valgrind.py:97 ../bin/apport-valgrind.py:97
#, python-format
msgid "Error: %s is not an executable. Stopping."
msgstr ""

#: ../kde/apport-kde.py:300 ../debian/tmp/usr/share/apport/apport-kde.py:300
msgid "Username:"
msgstr ""

#: ../kde/apport-kde.py:301 ../debian/tmp/usr/share/apport/apport-kde.py:301
msgid "Password:"
msgstr ""

#: ../kde/apport-kde.py:391 ../debian/tmp/usr/share/apport/apport-kde.py:391
msgid "Collecting Problem Information"
msgstr "Prikupljanje Informacija o Problemu"

#: ../kde/apport-kde.py:393 ../debian/tmp/usr/share/apport/apport-kde.py:393
msgid ""
"The collected information can be sent to the developers to improve the "
"application. This might take a few minutes."
msgstr ""

#: ../kde/apport-kde.py:419 ../debian/tmp/usr/share/apport/apport-kde.py:419
msgid "Uploading Problem Information"
msgstr "Prenos Informacija o Problemu"

#: ../kde/apport-kde.py:421 ../debian/tmp/usr/share/apport/apport-kde.py:421
#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:17 ../gtk/apport-gtk.ui.h:17
msgid ""
"The collected information is being sent to the bug tracking system. This "
"might take a few minutes."
msgstr ""
"Prikupljene informacije se šalju sustavu za praćenje grešaka. Ovo može "
"potrajati nekoliko minuta."

#: ../kde/apport-kde.py:455 ../kde/apport-kde.py:490
#: ../debian/tmp/usr/share/apport/apport-kde.py:455
#: ../debian/tmp/usr/share/apport/apport-kde.py:490
#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:1 ../gtk/apport-gtk.ui.h:1
msgid "Apport"
msgstr "Apport"

#: ../apport/com.ubuntu.apport.policy.in.h:1
msgid "Collect system information"
msgstr ""

#: ../apport/com.ubuntu.apport.policy.in.h:2
msgid ""
"Authentication is required to collect system information for this problem "
"report"
msgstr ""

#: ../apport/com.ubuntu.apport.policy.in.h:3
msgid "System problem reports"
msgstr ""

#: ../apport/com.ubuntu.apport.policy.in.h:4
msgid ""
"Please enter your password to access problem reports of system programs"
msgstr ""

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:2 ../gtk/apport-gtk.ui.h:2
msgid "Cancel"
msgstr ""

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:3 ../gtk/apport-gtk.ui.h:3
msgid "OK"
msgstr ""

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:4 ../gtk/apport-gtk.ui.h:4
msgid "Crash report"
msgstr ""

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:5 ../gtk/apport-gtk.ui.h:5
msgid "<big><b>Sorry, an internal error happened.</b></big>"
msgstr ""

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:7 ../gtk/apport-gtk.ui.h:7
msgid "Send an error report to help fix this problem"
msgstr ""

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:8 ../gtk/apport-gtk.ui.h:8
msgid "Ignore future problems of this program version"
msgstr ""

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:10 ../gtk/apport-gtk.ui.h:10
msgid "_Examine locally"
msgstr ""

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:13 ../gtk/apport-gtk.ui.h:13
msgid "<big><b>Collecting problem information</b></big>"
msgstr "<big><b>Sakupljanje podataka o problemu</b></big>"

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:14 ../gtk/apport-gtk.ui.h:14
msgid ""
"Information is being collected that may help the developers fix the problem "
"you report."
msgstr ""
"Skupljaju se informacije koje mogu pomoći razvojnom timu pri rješavanju "
"problema koje prijavljujete."

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:16 ../gtk/apport-gtk.ui.h:16
msgid "<big><b>Uploading problem information</b></big>"
msgstr "<big><b>Učitavanje informacija o problemu</b></big>"

#: ../kde/apport-kde-mimelnk.desktop.in.h:1
msgid "Apport crash file"
msgstr "Apport datoteka o rušenju"

#: ../bin/apport-unpack.py:22 ../debian/tmp/usr/bin/apport-unpack.py:22
#, python-format
msgid "Usage: %s <report> <target directory>"
msgstr ""

#: ../bin/apport-unpack.py:46 ../debian/tmp/usr/bin/apport-unpack.py:46
msgid "Destination directory exists and is not empty."
msgstr "Odredišni direktorji postiji i nije prazan."

#~ msgid "You can help the developers to fix the problem by reporting it."
#~ msgstr "Možete pomoći razvojnom timu da riješi problem prijavom istog."

#, python-format
#~ msgid "The package \"%s\" failed to install or upgrade."
#~ msgstr "Paket \"%s\" se nije uspio instalirati ili nadograditi"

#, no-c-format, python-format
#~ msgid "Complete report (recommended; %s)"
#~ msgstr "Potpun izvještaj (preporučeno; %s)"

#~ msgid ""
#~ "<big><b>Send problem report to the developers?</b></big>\n"
#~ "\n"
#~ "After the problem report has been sent, please fill out the form in the "
#~ "automatically opened web browser."
#~ msgstr ""
#~ "<big><b>Poslati izvještaj o problemu razvojnom timu?</b></big>\n"
#~ "\n"
#~ "Nakon što je izvještaj o problemu poslan, molim ispunite formular u "
#~ "automatski otvorenom internet pregledniku."

#~ msgid ""
#~ "If you were not doing anything confidential (entering passwords or other "
#~ "private information), you can help to improve the application by reporting "
#~ "the problem."
#~ msgstr ""
#~ "Ako niste radili nešto povjerljivo (unosili lozinke ili nešto slično), "
#~ "možete pomoći u poboljšanju aplikacije prijavljivanjem problema razvojnom "
#~ "timu."

#~ msgid "Application problem"
#~ msgstr "Problem s aplikacijom"

#~ msgid "_Send Report"
#~ msgstr "_Pošalji izvješće"

#~ msgid ""
#~ "If you were not doing anything confidential (entering passwords or other\n"
#~ "private information), you can help to improve the application by reporting\n"
#~ "the problem."
#~ msgstr ""
#~ "Ako niste radili ništa povjerljivo (unosili lozinke ili druge\n"
#~ "privatne informacije), možete pomoći unaprijediti aplikaciju prijavom\n"
#~ "problema."

#, python-format
#~ msgid "%s closed unexpectedly on %s at %s."
#~ msgstr "%s neočekivano prekinut na %s u %s."

#~ msgid "Restart _Program"
#~ msgstr "Ponovo pokreni _Program"

#, no-c-format
#~ msgid ""
#~ "<span weight=\"bold\" size=\"larger\">Sorry, the package \"%s\" failed to "
#~ "install or upgrade.</span>"
#~ msgstr ""
#~ "<span weight=\"bold\" size=\"larger\">Nažalost, paket \"%s\" se nije uspio "
#~ "instalirati ili nadograditi</span>"

#, python-format
#~ msgid "&Send complete report (recommended; %s)"
#~ msgstr "&Pošaljite potpun izvještaj (preporučeno; %s)"

#, no-c-format, python-format
#~ msgid "Reduced report (slow Internet connection; %s)"
#~ msgstr "Sažeti izvještaj (spora Internet veza; %s)"

#~ msgid "_Ignore future crashes of this program version"
#~ msgstr "_Zanemari buduća rušenja ove verzije programa"

#, python-format
#~ msgid "Send &reduced report (slow Internet connection; %s)"
#~ msgstr "Pošaljite &sažeti izvještaj (spora Internet veza; %s)"

#~ msgid ""
#~ "This will remove some large items from the report. These are very useful for "
#~ "developers to debug the problem, but might be too big for you to upload if "
#~ "you have a slow internet connection."
#~ msgstr ""
#~ "Ovo će odstraniti neke veće dijelove izvještaja. Oni su vrlo korisni "
#~ "razvojnom timu za otklanjanje problema, ali mogu biti preveliki za slanje "
#~ "ako imate sporu vezu s internetom."

#~ msgid "&Report Problem..."
#~ msgstr "&Prijavi Problem..."

#~ msgid "_Report Problem..."
#~ msgstr "_Prijavi problem..."

#~ msgid "Kernel problem"
#~ msgstr "Problem s kernelom"

#~ msgid ""
#~ "You can help the developers to fix the package by reporting the problem."
#~ msgstr ""
#~ "I vi možete pomoći razvojnom timu da popravi paket prijavom problema."

#, python-format
#~ msgid "This is not a genuine %s package"
#~ msgstr "Ovo nije izvorni %s paket"

#~ msgid "Content of the report"
#~ msgstr "Sadržaj izvještaja"

#~ msgid "Package problem"
#~ msgstr "Problem s paketom"

#~ msgid "incorrect number of arguments; use --help for a short help"
#~ msgstr ""
#~ "Netočan broj argumenata; upotrijebite --pomoć(ili --help) za kratku pomoć."
